<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WelcomeView</class>
 <widget class="QScrollArea" name="WelcomeView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1011</width>
    <height>803</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Welcome</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="widgetResizable">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="scrollAreaWidgetContents">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1011</width>
     <height>803</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="layoutRoot" rowstretch="3,5,3" columnstretch="3,5,3">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="1">
     <spacer name="spacer6">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <spacer name="spacer4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <widget class="QFrame" name="frameContent">
      <layout class="QVBoxLayout" name="layoutContent" stretch="0,1,0">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <widget class="QWidget" name="widgetHeader" native="true">
         <layout class="QHBoxLayout" name="layoutHeader" stretch="1,0,0,1">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="spacer1">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item alignment="Qt::AlignVCenter">
           <widget class="QLabel" name="labelIcon"/>
          </item>
          <item alignment="Qt::AlignVCenter">
           <widget class="QWidget" name="widgetAbout" native="true">
            <layout class="QVBoxLayout" name="verticalAbout">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="labelTitle"/>
             </item>
             <item>
              <widget class="QLabel" name="labelDescription"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="spacer2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layoutPanels" stretch="1,0">
         <property name="spacing">
          <number>20</number>
         </property>
         <item>
          <widget class="QWidget" name="widgetRecentFiles" native="true">
           <layout class="QGridLayout" name="layoutRecentFiles" rowstretch="0,0,0,1" columnstretch="0,1,0">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="horizontalSpacing">
             <number>20</number>
            </property>
            <item row="2" column="0">
             <widget class="QPushButton" name="buttonOpenArchive">
              <property name="text">
               <string>Open Archive...</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="buttonNewArchive">
              <property name="text">
               <string>New Archive</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="buttonClearRecentFiles">
              <property name="toolTip">
               <string>Clear Recent Files</string>
              </property>
              <property name="icon">
		      <iconset theme="edit-clear-all"/>
              </property>
             </widget>
            </item>
            <item row="1" column="1" rowspan="3" colspan="2">
             <widget class="QListView" name="listViewRecentFiles">
              <property name="minimumSize">
               <size>
                <width>400</width>
                <height>200</height>
               </size>
              </property>
              <property name="contextMenuPolicy">
               <enum>Qt::CustomContextMenu</enum>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="labelRecentFiles">
              <property name="text">
               <string>Recent Files</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <spacer name="spacer3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widgetHelp" native="true">
           <layout class="QVBoxLayout" name="layoutHelp" stretch="0,0,0,1">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="labelHelp">
              <property name="text">
               <string>Help</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignLeft">
             <widget class="KUrlLabel" name="labelHomepage">
              <property name="text">
               <string>Visit homepage</string>
              </property>
              <property name="url">
               <string/>
              </property>
              <property name="glowEnabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignLeft">
             <widget class="KUrlLabel" name="labelHandbook">
              <property name="text">
               <string>Read handbook</string>
              </property>
              <property name="url">
               <string/>
              </property>
              <property name="glowEnabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer11">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="widgetFooter" native="true">
         <layout class="QHBoxLayout" name="layoutFooter">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="spacer9">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxShowOnStartup">
            <property name="text">
             <string>Show on startup</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="spacer10">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="2">
     <spacer name="spacer5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <spacer name="spacer7">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2" alignment="Qt::AlignRight|Qt::AlignTop">
     <widget class="QWidget" name="widgetClose" native="true">
      <layout class="QHBoxLayout" name="layoutClose">
       <item>
        <widget class="QPushButton" name="buttonClose">
         <property name="toolTip">
          <string>Close Welcome Page</string>
         </property>
         <property name="icon">
		 <iconset theme="window-close"/>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlLabel</class>
   <extends>QLabel</extends>
   <header>kurllabel.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>buttonNewArchive</tabstop>
  <tabstop>buttonOpenArchive</tabstop>
  <tabstop>listViewRecentFiles</tabstop>
  <tabstop>buttonClearRecentFiles</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
